invenio_communities/members/services/schemas.py

```diff
@@ -140,7 +140,8 @@ def get_group_member(self, group):
         return {
             "type": "group",
             "id": group["id"],
-            "name": group["id"] or "",
+            "name": group.get("name") or group["id"],
```

Comment:
is the name optional or mandatory in the DB?

Reply:
Name is optional [see here](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/models.py#L69)